omero_marshal/__init__.py

```diff
@@ -53,7 +53,7 @@ def get_schema_version(version):
     if m is None:
         raise Exception("Invalid OMERO version number: " + version)
     v = StrictVersion(m.group(1))
-    if v >= StrictVersion('5.1.0') and v < StrictVersion('5.3.0'):
```

Comment:
Although OMERO 5.1 is no longer supported by the library, was there any issue associated with the mapping between the OMERO version and the underlying OME model version?